translations/harbour-osmscout-server-de.ts

```diff
@@ -0,0 +1,934 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+<context>
+    <name>AboutPage</name>
+    <message>
+        <location filename="../qml/pages/AboutPage.qml" line="8"/>
+        <source>The server has to be used with the client program that is able to display maps, query for searches and display selected routes. The server is a thin layer exposing functionality of &lt;i&gt;libosmscout&lt;/i&gt;, &lt;i&gt;libpostal&lt;/i&gt; and &lt;i&gt;geocoder-nlp&lt;/i&gt; through http. This server can be used as a drop-in replacement for online services providing maps.</source>
+        <translation>Der Server muss mit dem Client-Programm verwendet werden, das Karten, Suchanfragen und ausgew�hlte Routen anzeigt werden k�nnen. Der Server ist sehr schlank aus der Funktionalit�t von & lt; i & gt; libosmscout & lt; / i & gt; libpostal & lt; / i & gt; Und & lt; i & rdquo; geocoder-nlp & lt; / i & gt; �ber http. Dieser Server kann als Drop-in-Ersatz f�r Online-Dienste mit Karten verwendet werden.</translation>
```

Comment:
Here it looks that Qt .ts files are using special symbols for making some words _italic_. In the translated version, I can see the spaces between & and lt;, for example. But I would remove them after merging, not a big deal :)